bedrock/base/tests/test_accepted_locales.py

```diff
@@ -65,7 +65,7 @@ def test_build_dev_languages(self):
         """
         settings.DEV = True
         langs = get_dev_languages()
-        assert (langs == ['en-US', 'fr'] or langs == ['fr', 'en-US'],
+        assert langs == ['en-US', 'fr'] or langs == ['fr', 'en-US'], (
```

Comment:
yeah, I missed this in the review :(
flake8 isn't complaining about this version.